x-pack/plugins/ml/public/application/explorer/alerts/alerts_panel.tsx

```diff
@@ -0,0 +1,113 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import {
+  EuiButtonGroup,
+  EuiFlexGroup,
+  EuiFlexItem,
+  EuiNotificationBadge,
+  EuiSpacer,
+  EuiLoadingSpinner,
+} from '@elastic/eui';
+import { i18n } from '@kbn/i18n';
+import { FormattedMessage } from '@kbn/i18n-react';
+import { AlertConsumers } from '@kbn/rule-data-utils';
+import React, { type FC, useState } from 'react';
+import useObservable from 'react-use/lib/useObservable';
+import { ML_ALERTS_CONFIG_ID } from '../../../alerting/anomaly_detection_alerts_table/register_alerts_table_configuration';
+import { CollapsiblePanel } from '../../components/collapsible_panel';
+import { useMlKibana } from '../../contexts/kibana';
+import { useAnomalyExplorerContext } from '../anomaly_explorer_context';
+import { AlertsSummary } from './alerts_summary';
+import { AnomalyDetectionAlertsOverviewChart } from './chart';
+import { statusNameMap } from './const';
+
+export const AlertsPanel: FC = () => {
+  const {
+    services: { triggersActionsUi },
+  } = useMlKibana();
+
+  const [isOpen, setIsOpen] = useState(true);
+  const [toggleSelected, setToggleSelected] = useState(`alertsSummary`);
+
+  const { anomalyDetectionAlertsStateService } = useAnomalyExplorerContext();
+
+  const countByStatus = useObservable(anomalyDetectionAlertsStateService.countByStatus$);
+  const alertsQuery = useObservable(anomalyDetectionAlertsStateService.alertsQuery$, {});
+  const isLoading = useObservable(anomalyDetectionAlertsStateService.isLoading$, true);
+
+  const alertStateProps = {
+    alertsTableConfigurationRegistry: triggersActionsUi!.alertsTableConfigurationRegistry,
+    configurationId: ML_ALERTS_CONFIG_ID,
+    id: `ml-details-alerts`,
+    featureIds: [AlertConsumers.ML],
+    query: alertsQuery,
+    showExpandToDetails: true,
+    showAlertStatusWithFlapping: true,
+  };
+  const alertsStateTable = triggersActionsUi!.getAlertsStateTable(alertStateProps);
+
+  const toggleButtons = [
+    {
+      id: `alertsSummary`,
+      label: i18n.translate('xpack.ml.explorer.alertsPanel.summaryLabel', {
+        defaultMessage: 'Summary',
+      }),
+    },
+    {
+      id: `alertsTable`,
+      label: 'Details',
```

Comment:
"Details" needs to be internationalized

Reply:
Added translation in 587ae077798af0b1c4ddf067354e21f69f689656